### ISSUE TITLE
[Misc] Training dummies

### ISSUE THREAD

**Issue by natrow** [natrow/tfc_minecolonies#19]
[Minecolonies' wiki](https://wiki.minecolonies.ldtteam.com/source/buildings/combatacademy) says certain military buildings require training dummies made from a pumpkin and hay bale. This functionality needs to be verified against TFC blocks.

**Comment by Chumpy83** [comment]
There's no related pumpkin item outside of the add-on Firmalife. You can 100% carve one to be a vanilla jack-o-lantern (seems to be what the schematics actually use). Some schematics let you use the tag tool to mark the relevant points for work but doesn't seem to be the case for this one.

The vanilla hay_block seems to be craftable in plain TFC with a 3x3 of grass, so that block's effectively already set.

**Comment by natrow** [comment]
Good point, I think when I made this I was worried it used the actual pumpkin block (which rots in TFC). Does the TFC pumpkin carve into a vanilla pumpkin block or does TFC add its own?

**Comment by Chumpy83** [comment]
Wow SUPER delayed reply but - one Firmalife pumpkin 'carved' with the face turns into the 'minecraft:carved_pumpkin' - vanilla version (so Firmalife would end up being a requirement for the pumpkin). SCRATCH THAT - tfc:pumpkin is the used item, the carving recipe is from Firmalife though.
